src/components/BillCard/SmallBillCard/styles.scss

```diff
@@ -1,57 +1,51 @@
 @import "../../../variables";
 
-div.bill-active {
```

Comment:
in this case, the `-` is ok because it's just a color change. `bill-active` should just be a subset of the `bill`